identity/src/evm/persistent.rs

```diff
@@ -49,10 +49,11 @@ impl<T: Clone + Eq + Hash + AsRef<[u8]> + TryFrom<KeyInfo>> KeyStore for Persist
         self.memory.list()
     }
 
-    fn put(&mut self, info: KeyInfo) -> Result<()> {
-        self.memory.put(info)?;
+    fn put(&mut self, info: KeyInfo) -> Result<Self::Key> {
+        let addr = self.memory.put(info)?;
         // TODO: We can flush to disk only after certain number of `put` is called.
```

Comment:
Would you mind explaining me what do you mean with this TODO?

Reply:
I mean to flush to disk only after some buffer mechanism, so that we dont have to flush every time when we call put.